docs/icon.md

```diff
@@ -0,0 +1,28 @@
+# Icon
+### Usage
+
+```jsx
+import { Icon } from 'radiance-ui';
+
+<Icon width={14} height={14} iconName="camera" />
```

Comment:
Icons _should_ all be 16x16 by default. I don't know what this looks like everywhere else. 

Might be worth a look but I am considering adding a smaller size in the future.